### PR TITLE
Getting started with docker

### PR DESCRIPTION
**Pull request by csquared**
Shows how to create a simple static html app from a `Dockerfile` and `docker-compose.yml` using the `httpd` image.

### CODE REVIEW

File: _languages_and_frameworks/getting-started-with-docker.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+Your `Dockerfile` should look like this:
+
+    FROM httpd
+
+And your `docker-compose.yml` should look like this:
+
+    main:
+      build: .
+      ports:
+        - 5000:80
```
Comment (ddollar) [inline _languages_and_frameworks/getting-started-with-docker.md]: These should be reversed

Comment (ddollar) [inline _languages_and_frameworks/getting-started-with-docker.md]: Actually for httpd it should just be 80:89

Comment (ddollar) [inline _languages_and_frameworks/getting-started-with-docker.md]: 80:80 even

Comment (csquared) [inline _languages_and_frameworks/getting-started-with-docker.md]: changed

File: _languages_and_frameworks/getting-started-with-docker.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    FROM httpd
+
+And your `docker-compose.yml` should look like this:
+
+    main:
+      build: .
+      ports:
+        - 80:80
+
+Convox uses Docker under the hood for containerization, and these two files contain all the information it needs to build and run your app.
```
Comment (nzoschke) [inline _languages_and_frameworks/getting-started-with-docker.md]: This example is fine, but you could make it even shorter by saying `image: httpd` in docker-compose.yml and using no Dockerfile!

One of my guiding principles is all things equal, the shortest solution wins. But your taste may vary :)

Comment (nzoschke) [inline _languages_and_frameworks/getting-started-with-docker.md]: Ah, I jumped the gun. The subsequent COPY necessitates a Dockerfile

File: _languages_and_frameworks/getting-started-with-docker.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      ports:
+        - 80:80
+      volumes:
+        - ./public-html/:/usr/local/apache2/htdocs/
+
+Restart your application with
+
+    $ convox start
+
+Now when you change files in the `./public-html` directory of your local filesystem the changes are automatically synced to your development containr.
```
Comment (nzoschke) [inline _languages_and_frameworks/getting-started-with-docker.md]: container

**Comment by nzoschke** [comment]
:+1: Love using the `httpd` image. It works! gives me warm fuzzies.

**Pull request merged**